src/StatusMsg.chpl

```diff
@@ -0,0 +1,24 @@
+module StatusMsg {
+    use Reflection;
+    use ServerConfig;
+    use Logging;    
+    use Message;
+    use MemoryMgmt;
+    use MultiTypeSymbolTable;
+    use MultiTypeSymEntry;
+    
+    private config const logLevel = ServerConfig.logLevel;
+    private config const logChannel = ServerConfig.logChannel;
+    const sLogger = new Logger(logLevel, logChannel);
+    
+    /**
+     * 
+     */
```

Comment:
Since this isn't user facing, I don't think it's terribly important to have doc strings. I'm fine with adding them or just deleting the block comment. Adding a suggestion for convenience in case u decide to go the "just delete" path
```suggestion
```